README.md

```diff
@@ -68,36 +68,39 @@ composite(tiles, zxy, options, function(err, result) {
 
 ### `localize`
 
-Modify a tile's features and properties to support localized languages and worldviews. This function requires the input vector tiles to match a specific schema for language translation and worldviews.
+A filtering function for modifying a tile's features and properties to support localized languages and worldviews. This function requires the input vector tiles to match a specific schema for language translation and worldviews.
 
 #### Parameters
 
-- `buffer` **Buffer** a vector tile buffer, gzip compressed or not
-- `language` **String** the IETF BCP 47 language code.
-- `worldview` **String** a country code used to filter features. A feature will be processed into worldviews if it has a `_mbx_worldview` property. The `_mbx_worldview` property must contain a comma-separated string of ISO 3166-1 alpha-2 country codes that define which worldviews the feature represents (example: `US,RU,IN`).
-  - vtcomposite will determine if a feature should be included based on the provided `worldview` value and any `_mbx_worldview` property on that feature.
-    - Match: the feature is cloned and `worldview: XX` is added to the feature's properties and the `_mbx_worldview` property is dropped. If the original feature contains a `worldview` property, it is overwritten.
-    - No match: the entire feature is dropped.
-  - Given a null `worldview` parameter and a feature with an `_mbx_worldview` property, four "legacy" worldviews `US,CN,IN,JP` are used to match. A single `_mbx_worldview` feature may be split into a maximum of four worldviews in this case.
-  - Any feature _without_ an `_mbx_worldview` property is retained.
-- `options` **Object** _(optional)_
-  - `options.compress` **Boolean** a boolean value indicating whether or not to return a compressed buffer. Default is to return an uncompressed buffer. (optional, default `false`)
+- `params` **Object**
+  - `params.buffer` **Buffer** a vector tile buffer, gzip compressed or not
+  - `params.compress` **Boolean** a boolean value indicating whether or not to return a compressed buffer. Default is to return an uncompressed buffer. (optional, default `false`)
+  - `params.language` **String** the IETF BCP 47 language code.
+  - `params.worldview` **Array<String>** array of ISO 3166-1 alpha-2 country codes used to filter out features of different worldviews. Worldview data must be included in the vector tile. See `params.worldview_property` for more details on encoding data.
+    - If a feature matches the requested worldview the feature is cloned and `worldview: XX` is added to the feature's properties and the `params.worldview_property` property is dropped. If the original feature contains a `worldview` property, it is overwritten.
```

Comment:
IDK. This may be clearer?

```suggestion
    - If a feature matches one of the requested worldviews, the feature is kept. It will have a property `worldview` equal to the matching worldview value and the `params.worldview_property` property will be dropped. If the original feature contained a `worldview` property, it is overwritten.
```